content/tracing/setup/javascript.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Tracing JavaScript Applications
+kind: Documentation
+aliases:
+- /tracing/javascript/
+- /tracing/languages/javascript/
+further_reading:
+- link: "https://github.com/DataDog/dd-trace-js"
+  tag: "Github"
+  text: Source code
+- link: "tracing/visualization/"
+  tag: "Use the APM UI"
+  text: "Explore your services, resources and traces"
+---
+
+<div class="alert alert-warning">
+JavaScript APM is currently in <strong>beta</strong>.
+</div>
+
+## Getting started
+
+For descriptions of terminology used in APM, take a look at the [official documentation][visualization docs].
+
+For details about contributing, check out the [development guide][development docs].
+
+[visualization docs]: https://docs.datadoghq.com/tracing/visualization/
+[development docs]: https://github.com/DataDog/dd-trace-js/blob/master/README.md#development
+
+### Installation
+
+To begin tracing Node.js applications, first [install and configure the Datadog Agent](/tracing/setup/) (see additional documentation for [tracing Docker applications](/tracing/setup/docker/)).
+
+Next, install the Datadog Tracing library using npm:
+
+```sh
+npm install --save dd-trace
+```
+
+Finally, import the tracer and instrument your code!
+
+### Example
```

Comment:
I would call this section "Manual Instrumentation" adding this incipit:

If you aren’t using supported library instrumentation (see Library compatibility), you may want to to manually instrument your code. Adding tracing to your code is easy using the tracing method, which you can wrap around your JavaScript code. The following example, initializes a Datadog Tracer and creates a `Span` called `web.request`:

(your code)